provider/scripts/spark/offline_store_spark_runner.py

```diff
@@ -7,7 +7,7 @@
 import sys
 import types
 import uuid
-from datetime import datetime
+import datetime
```

Comment:
need access to `isinstance`

Reply:
why not keep it `from datetime import datetime`?

Reply:
unfortunately that causes `datetime.date` to refer to a method of the `datetime` class (which doesn't work with `isinstance`)

so to fix you instead pull the module, so then `datetime.date` refers to the class, and `isinstance` will work.

Reply:
I always forget that and keep re-googling to the same stack post lol 

https://stackoverflow.com/questions/5665597/python-isinstance-returning-error-with-datetime-date